### PR TITLE
Fix issue preventing OData Connected Service from being listed in VS

### LINKED ISSUE #172: OData Connected Service v. 0.11 does not appear in the Connected Service list
### Describe the bug
I installed the "OData Connected Service" extension via the Extensions-> Manage Extensions menu (I also tried to download and install the extension from this github repository). After restarting Visual Studio, I tried to make a service by selecting Add -> Connected Service from the project context menu (Net Core 3.1 console app). But it is not on the list of supported services.
At the same time, version 0.10 is normally displayed in the list after installation and allows you to create a proxy class

#### Version of the Project affected
ODataConnectedService v0.11
VisualStudio Version v16.6.2
Microsoft.Odata.Client v7.7.2

#### To Reproduce
1. Install OData Connected Service from marketplace or this repo
2. Restart VS
3. Right click on the project and select Add->Connected Service menu item
4. OData Connected Service not showing in Other Services list

#### Expected behavior
The Connected OData Service extension should appear in the Other Services list

#### Actual behavior
The Connected OData Service extension version 0.11 does not appear in the Other Services list.

#### Screenshots
Screenshot_1

### PR DESCRIPTION
**Pull request by habbes**
Fix #172 

The connected service could not be loaded by Visual Studio because the `Microsoft.OData.ConnectedService.dll` inside the package was not properly signed.

Running `sn.exe -v Microsoft.OData.ConnectedService.dll` reported that it was delay-signed assembly. This means that the assembly still needs to be signed, indicating that the signing tool used in the build pipeline was not correctly signing the assembly.

The fix consists in:
- Disabling delay-signing
- Replacing the Microsoft-owned strong name public key file with a new strong name key-pair (including both public and private key, see the note on .NET core projects here: https://docs.microsoft.com/en-us/dotnet/standard/assembly/strong-named)
- Simplifying the build pipeline

Now running `sn.exe -v Microsoft.OData.ConnectedService.dll` reports that the assembly is valid.

### CODE REVIEW

File: src/source.extension.vsixmanifest [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
     <Metadata>
-        <Identity Id="ODataConnectedService.e30335d6-f9c7-4d08-b66a-f011f3f18477" Version="0.11.0" Language="en-US" Publisher="Microsoft" />
+        <Identity Id="ODataConnectedService.e30335d6-f9c7-4d08-b66a-f011f3f18477" Version="0.12.0" Language="en-US" Publisher="Microsoft" />
```
Comment (gathogojr) [inline src/source.extension.vsixmanifest:4]: @habbes Is the publisher Microsoft due to something pending?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -153,42 +116,14 @@ stages:
...
-              ArtifactName: 'VSIX'
-              publishLocation: 'Container'
-
+          
+          # Sign the vsix archive as well as files contained in the vsix that are listed in config/filelist.txt
+          # For more info about the SignClient tool, visit: https://github.com/dotnet/SignService
           - pwsh: |
               .\SignClient "Sign" `
-              --baseDirectory "$(Build.ArtifactStagingDirectory)/VSIX" `
+              --baseDirectory "$(Pipeline.Workspace)/VSIX" `
```
Comment (gathogojr) [inline azure-pipelines.yml:124]:
```suggestion
              --baseDirectory "$(Pipeline.Workspace)/SignedVSIX" `
```
?

File: azure-pipelines.yml [hunk 3]
```diff
@@ -82,51 +69,27 @@ stages:
...
-        publishLocation: 'Container'
+        OverWrite: true
 
     - task: PublishBuildArtifacts@1
-      displayName: 'Publish Artifact - VSIXStaging'
+      displayName: 'Publish Artifact - VSIX'
       inputs:
-        PathtoPublish: '$(Build.ArtifactStagingDirectory)\VSIXStaging'
-        ArtifactName: 'VSIXStaging'
+        PathtoPublish: '$(Build.ArtifactStagingDirectory)\VSIX'
```
Comment (gathogojr) [inline azure-pipelines.yml:91]:
```suggestion
        PathtoPublish: '$(Build.ArtifactStagingDirectory)\SignedVSIX'
```
?

**Review by gathogojr — APPROVED** [review]
Left a few comments... Otherwise looks okay to me

**Review by marabooy — APPROVED** [review]

**Pull request merged**